docs/usage/misc.rst

```diff
@@ -0,0 +1,168 @@
+      pdf_func = interp1d(x, pdf_x)
+
+      with open(path, "wb") as f:
+         if verbose:
+            print(f"Saving model to {f.name}")
+         pickle.dump(pdf_func, f)
+
+      return pdf_func
+
+
+With zfit
+~~~~~~~~~
+
+This should be called after you have fitted your model and you have a ``zfit.pdf.BasePDF`` object.
+
+.. code-block:: python
+
+    from scipy.interpolate import interp1d
+    import pickle
+
+    def save_pdf(var, pdf, path="pdf.pkl", npoints=10000, verbose=True):
+        """
+        Save a PDF from zfit as a callable function.
+
+        Parameters
+        ----------
+        var : zfit.Space
+            The variable to evaluate the PDF at.
+        pdf : zfit.pdf.BasePDF
+            The PDF to save.
+        path : str, optional
+            The path to save the PDF to. Default is "pdf.pkl".
+        npoints : int, optional
+            The number of points to evaluate the PDF at. Default is 10000.
+        verbose : bool, optional
+            Whether to print progress. Default is True.
+
+        Returns
+        -------
+        pdf_func : scipy.interpolate.interp1d
+
+        Notes
+        -----
+        The PDF is saved as a scipy.interpolate.interp1d function with pickle.
+        """
+
+        lower, upper = var.limits
+        x = np.linspace(lower[-1][0], upper[0][0], npoints)
+
+        # Evaluate the PDF at the given points
+        pdf_x = zfit.run(pdf.pdf(x, norm_range=var))
+
+        # Interpolate the PDF
+        pdf_func = interp1d(x, pdf_x)
+
+        with open(path, "wb") as f:
+            if verbose:
+                print(f"Saving model to {f.name}")
+            pickle.dump(pdf_func, f)
+
+        return pdf_func
+
+
+Renormalize the PDF
+-------------------
+
+The ``pdf_func`` you get, by either get it from function or read the saved pickle file for ``RooFit`` or ``zfit``, has an area of 1. When you want to plot it, you need to multiply it by the bin width of your histogram, the number of expected events in the range and the integral of the PDF in the range. This can be done easily using this small function:
+
+.. code-block:: python
+
+   from scipy.integrate import quad
+
+   def renormalize(pdf, x_range, bins, n_data):
+      """
+      Renormalize a PDF to the number of data events.
+
+      Parameters
+      ----------
+      pdf : callable
+         The PDF to renormalize.
+      x_range : tuple
+         The range of the PDF.
+      bins : int
+         The number of bins.
+      n_data : int
+         The number of data events expected in the PDF range.
+
+      Returns
+      -------
+      pdf : callable
+         The renormalized PDF.
+      """
+      xmin, xmax = x_range
+      bin_width = (xmax - xmin) / bins
+      integral = quad(pdf, xmin, xmax)[0]
+
+      return lambda x: pdf(x) * n_data * integral * bin_width
+
+Then you can use ``plot_model()`` or ``plot_data_model_comparison()`` to plot the PDF and do all sort of comparisons with the ``plothist`` interface.
```

Comment:
* add a link to the documentation explaining how to plot a pull plot with plothist and model made of functions, maybe showing an example of plot here for illustration (but no RooFit plot in my opinion)
* add a link from the documentation explaining how to do a pull plot to this page.